webapp/jira.py

```diff
@@ -313,6 +313,27 @@ def unlink_parent_issue(self, issue_id: str) -> bool:
             data=payload,
         )
 
+    def link_copydoc_with_content_page(self, copydoc, jira_id):
+        """Link a copydoc with a content page in Jira.
+
+        Args:
+            copydoc (str): The URL of the copydoc.
+            jira_id (str): The ID of the Jira issue.
+
+        Returns:
+            dict: The response from the Jira API.
+        """
+
+        payload = {
+            "fields": {
+                "customfield_11133": copydoc,
```

Comment:
this field ID will change after we move to a real Content Team project in Jira, so worth adding a TODO comment here